### PR TITLE
updates to spooky

### PR DESCRIPTION
**Pull request by kendonB**
Let's see how this goes with the coverage bot

I added a few more tests with the right number of bytes to get hit by the missed bits of the algorithm. I also hashed iris which hit some other bits of the algorithm.

Also added a copyright statement at the top of spooky_serialize.cpp

### CODE REVIEW

File: src/SpookyV2.cpp [hunk 1]
```diff
@@ -9,9 +9,11 @@
 //   July 30 2012: I reintroduced the buffer overflow
 //   August 5 2012: SpookyV2: d = should be d += in short hash, and remove extra mix from long hash
 //   Kendon Bell April 30 2019: Added counters to help with skipping first n bytes
+//   Kendon Bell May 04 2019: Deleted original source code that doesn't get used
 
 #include <memory.h>
 #include "SpookyV2.h"
+#include "R.h"
```
Comment (eddelbuettel) [inline src/SpookyV2.cpp]: Why ?

Comment (kendonB) [reply src/SpookyV2.cpp]: Mistake - I had it there to use Rprintf to see when the various bits of the algorithm got hit

Comment (eddelbuettel) [reply src/SpookyV2.cpp]: Just send a follow up commit.  If need be I merge it again.  (I am new to the PR into PR merge business ;-)  )

Comment (eddelbuettel) [reply src/SpookyV2.cpp]: Else I just do it in the final cleanup.

**Comment by kendonB** [comment]
do you want a PR for the last tweaks or will you do it? A couple of nocovs and removing the include R.h

**Comment by eddelbuettel** [comment]
We can review.  We probably both a few items.  Got to run an errand now -- so no rush.  Thanks for for #103 : additional tests instead of tags "almost never happens" :)

**Pull request merged**